lib/id3c/cli/command/geocode.py

```diff
@@ -304,7 +305,7 @@ def get_response_from_cache_or_geocoding(address: dict,
         response = cache[key]
         LOG.debug('Response found in cache.')
     except KeyError:
-        response = cache[key] = geocode_address(address)
+        response = cache[key] = geocode_address(address) or extract_address(address)
```

Comment:
Extract address also does geocoding, so it's weird to me that it's the fallback to something called `geocode_address`. I think design-wise it'd be nicer to incorporate this fallback into `geocode_address` itself or something?

Reply:
Yup! Done in rework